### PR TITLE
Fix command line to work in Windows without any external binary

### PR DESCRIPTION
**Pull request by jobedom**

### CODE REVIEW

**Comment by jobedom** [comment]
I also added a .sublime-commands file to make the plugin show up in the command palette (Ctrl+Shift+P).

File: GitCommitMsg.sublime-commands [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[{
+    "caption": "GitCommitMsg: Show commit info for current line",
```
Comment (thecodejunkie) [inline GitCommitMsg.sublime-commands]: This should probably read `Show commit history for selected lines`

**Comment by thecodejunkie** [comment]
Awesome :thumbsup:

**Comment by jobedom** [comment]
You're right, thecodejunkie. Changed.

**Comment by cbumgard** [comment]
Great fix

**Pull request merged**